docs/99-references/00-references.adoc

```diff
@@ -35,21 +35,45 @@ ATTENTION: labels have to be non-numeric.
 **B**
 
 - [[[bachmann,Bachmann 2000]]] Bachmann, F., L. Bass, et al.: Software Architecture Documentation in Practice. Software Engineering Institute, CMU/SEI-2000-SR-004.
+- [[[barbacci, Barbacci 2003]]] Barbacci, M.R., Ellison, R., et al.: Quality Attribute Workshops (QAWs), Third Edition. Software Engineering Institute, CMU/SEI-2003-TR-016
 - [[[bass,Bass 2003]]] Bass, L., Clements, P. und Kazman, R. (2003): Software Architecture in Practice. Addison-Wesley, Reading, Mass
+- [[[bass2006, Bass 2006]]] Bass, L., Nord, R., et al.: Risk Themes Discovered Through Architecture Evaluations. Software Engineering Institute, CMU/SEI-2006-TR-012
```

Comment:
Labels dürfen _keine_ Zahlen oder Sonderzeichen beinhalten, nur Buchstaben von a-z sind erlaubt. Du musst das z. B.  als `bassa`, `bassb` kennzeichnen. 

Das steht auch etwas weiter oben in den Remarks der Datei. 😉

Reply:
Scheinbar geht das inzwischen mit Zahlen.